cmd/manager/main.go

```diff
@@ -39,6 +40,7 @@ func init() {
 	flagset := flag.CommandLine
 	flagset.StringVar(&flagImage, "grafana-image", "", "Overrides the default Grafana image")
 	flagset.StringVar(&flagImageTag, "grafana-image-tag", "", "Overrides the default Grafana image tag")
+	flagset.StringVar(&flagPluginsInitContainerImage, "grafana-plugins-init-container-image", "", "Overrides the default Grafana Plugins Init Container image")
```

Comment:
Would it make sense to have another flag for the tag as well, like `grafana-plugins-init-container-image-tag`?